packages/virtual/libc/package.mk

```diff
@@ -11,7 +12,7 @@ PKG_DEPENDS_INIT="toolchain glibc:init"
 PKG_SECTION="virtual"
 PKG_LONGDESC="Meta package for installing various tools and libs needed for libc"
 
-if [ "$BOOTLOADER" = "bcm2835-bootloader" ]; then
+if [ "${TARGET_ARCH}" = "arm" ]; then
```

Comment:
`arm-mem` was tested only on A7 and A53, perhaps something like `&& [[ "$TARGET_CPU" = "cortex-a7" ]] || [[ "$TARGET_CPU" = *"cortex-a53" ]]` should be added here?

Reply:
In that case I'd rather just continue limiting it to RPi/RPi2 than get into the game of cherry picking support for specific ARM CPU implementations. If we start restricting this to specific implementations of ARM CPU then I would start to question the sense of including it at all (even for RPi/RPi2) - at that point it's just more trouble/hassle than it's worth (IMHO).